wetterdienst/dwd/radar/store.py

```diff
@@ -5,56 +5,64 @@
 
 from wetterdienst import TimeResolution
 from wetterdienst.dwd.metadata.datetime import DatetimeFormat
+from wetterdienst.dwd.radar.metadata import RadarParameter
 
 
-def store_radolan_data(
+def store_radar_data(
+    parameter: RadarParameter,
     date_time_and_file: Tuple[datetime, BytesIO],
     time_resolution: TimeResolution,
     folder: Union[str, Path],
-) -> None:
+) -> None:  # pragma: no cover
+    """
+    Stores a binary file of radar data locally.
+    """
 
     date_time, file = date_time_and_file
 
-    filepath = build_local_filepath_for_radolan(date_time, folder, time_resolution)
+    filepath = _build_local_filepath(parameter, date_time, folder, time_resolution)
 
     filepath.parent.mkdir(parents=True, exist_ok=True)
 
     with filepath.open("wb") as f:
         f.write(file.read())
 
-    # When the file has been written, reset seek pointer.
-    file.seek(0)
-
 
-def restore_radolan_data(
-    date_time: datetime, time_resolution: TimeResolution, folder: Union[str, Path]
-) -> BytesIO:
-    filepath = build_local_filepath_for_radolan(date_time, folder, time_resolution)
+def restore_radar_data(
+    parameter: RadarParameter,
+    date_time: datetime,
+    time_resolution: TimeResolution,
+    folder: Union[str, Path],
+) -> BytesIO:  # pragma: no cover
+    """ Opens downloaded radar data into a binary object"""
+    filepath = _build_local_filepath(parameter, date_time, folder, time_resolution)
 
     with filepath.open("rb") as f:
         file_in_bytes = BytesIO(f.read())
 
     return file_in_bytes
 
 
-def build_local_filepath_for_radolan(
-    date_time: datetime, folder: Union[str, Path], time_resolution: TimeResolution
+def _build_local_filepath(
```

Comment:
Output has to be Path, not Union[str, Path]

Reply:
Thanks. `radar/store.py` is currently completely defunct.

We will have to take care about that later, unless @gutzbenj might want to dedicate some time to that.

Reply:
I'd suggest we remove it for now and add the feature in a later iteration.

Reply:
Removed with f4f5de6c6.